deploy/acm-policies/50-GENERATED-osd-limited-support.Policy.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: osd-limited-support
+    namespace: openshift-limited-support-policies
+spec:
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: osd-limited-support
+            spec:
+                object-templates:
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: v1
+                        kind: ConfigMap
+                        metadata:
+                            name: limited-support
+                            namespace: openshift-osd-metrics
```

Comment:
What happens if this namespace isn't present on the HCP?

Reply:
Per https://coreos.slack.com/archives/CMD1L32M9/p1672776350672259 all policies will be in a single namespace moving forward. There will be an upcoming PR to change the name to something other than openshift-rbac-policies, so for now this will revert to openshift-rbac-policies.